apps/minifront/src/components/shared/asset-selector.tsx

```diff
@@ -3,18 +3,18 @@ import {
   DialogClose,
   DialogContent,
   DialogHeader,
-  DialogTrigger,
 } from '@penumbra-zone/ui/components/ui/dialog';
 import { AssetIcon } from '@penumbra-zone/ui/components/ui/tx/view/asset-icon';
 import {
   Metadata,
   ValueView,
 } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/asset/v1/asset_pb';
 import { ValueViewComponent } from '@penumbra-zone/ui/components/ui/tx/view/value';
-import { useEffect, useMemo, useState } from 'react';
+import { useEffect, useId, useMemo, useState } from 'react';
```

Comment:
All the changes in this file are related to making the modal appear to grow out of the button that's clicked. There are no functional/logical changes here.